test/test-source-map-consumer.js

```diff
@@ -1049,26 +1049,6 @@ exports["test sourceRoot + originalPositionFor"] = async function(assert) {
   map.destroy();
 };
 
-exports["test github issue #56"] = async function(assert) {
```

Comment:
It seems to me that this test should still work and should not be removed.

Reply:
I added a new `sourceRoot` example [here](https://tromey.github.io/source-map-examples/) that should show this in action.  Works in Chrome.

Reply:
It's a breaking change, you're not wrong, but it also seems like something that isn't necessary to support. I have a super hard time imagining a case where anyone is actually going to do this. Webpack does

```
sourceRoot: "webpack:///",
sources: ["/foo/bar.js"],
```
but that will work fine with the existing code. It seems very unlikely that someone would split up a scheme and a host into two separate pieces in a real map, and if they do that seems like something we should be discouraging.